src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,24 @@
+import { HStack, Flex } from "@chakra-ui/react"
+
+export interface FooterProps {
+  children: JSX.Element
+}
+
+export const Footer = ({ children }: FooterProps): JSX.Element => (
+  <Flex
+    h="80px"
+    w="100%"
+    pr="30px"
+    position="fixed"
+    background="background.action.defaultInverse"
+    border="1px"
+    borderColor="border.divider.alt"
+    justify="end"
+    align="center"
+    bottom="0"
+  >
+    <HStack>
+      <>{children}</>
```

Comment:
nit: don't need to wrap in fragment

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/932/commits/7faa682fd7af5140926947afef3520d12daa67fb